evap/staff/templates/staff_user_form.html

```diff
@@ -14,7 +14,22 @@
 
 {% block content %}
     {{ block.super }}
-    <h3>{% if form.instance.id %}{% trans 'Edit user' %}{% else %}{% trans 'Create user' %}{% endif %}</h3>
+
+    <div class="d-flex mb-3">
+        <h3>{% if form.instance.id %}{% trans 'Edit user' %}{% else %}{% trans 'Create user' %}{% endif %}</h3>
+        {% if form.instance.id %}
+            <div class="ms-auto d-print-none">
+                {% if has_due_evaluations %}
+                    <button type="button" class="btn btn-sm btn-light" onclick="resendEmailModalShow({{ form.instance.id }}, '{{ form.instance.full_name|escapejs }}');">{% trans 'Resend evaluation started email' %}</button>
+                {% else %}
+                    <div title="This user currently has no due evaluations." data-bs-toggle="tooltip" data-bs-placement="bottom">
+                        <button disabled type="button" class="btn btn-sm btn-light">{% trans 'Resend evaluation started email' %}</button>
+                    </div>
+                {% endif %}
```

Comment:
In one branch we have a div and a button and in the other only a button - let's have the div in both just to be sure that the layout is the same in both versions

Reply:
In which other branch do we have another div?

Reply:
In case `has_due_evaluations` is true, we insert a button, but if `has_due_evaluations` is false, a div containing a button is inserted into the HTML. To ensure that both cases look the same, we should also include a div in the first case, even if it doesn't seems unneeded:
```suggestion
                {% if has_due_evaluations %}
                    <div>
                        <button type="button" class="btn btn-sm btn-light" onclick="resendEmailModalShow({{ form.instance.id }}, '{{ form.instance.full_name|escapejs }}');">{% trans 'Resend evaluation started email' %}</button>
                    </div>
                {% else %}
                    <div title="This user currently has no due evaluations." data-bs-toggle="tooltip" data-bs-placement="bottom">
                        <button disabled type="button" class="btn btn-sm btn-light">{% trans 'Resend evaluation started email' %}</button>
                    </div>
                {% endif %}
```